client/src/views/EventsView.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <v-row class="bg-primaryWhite pl-4 pt-4" align="center" no-gutters>
+    <v-col>
+      <v-text-field
+      prepend-inner-icon="mdi-magnify"
+      hide-details
+      placeholder="Search Events"
+      clearable
+      v-model="searchQuery"
+    />
+    </v-col>
+    <v-col class="justify-center d-flex" cols="2">
+      <v-btn
+        v-if="tempIsTeamAdmin"
+        size="x-large"
+        density="compact"
+        icon="mdi-plus"
+        class="bg-primaryRed text-primaryWhite"
+        @click="isAddingEvent = true"
+      >
+      </v-btn>
+    </v-col>
+  </v-row>
+  <div id="cards-container" class="bg-primaryWhite pt-4">
+    <Event
+      v-for="event in filteredEventsList"
+      key="event.id"
+      :event="event"
+      :isTeamAdmin="tempIsTeamAdmin"
+      @edit="openEditModal"
+    />
+  </div>
+  <EventsModal v-if="isAddingEvent" :type="'Create'" @close="closeModal" v-model="isAddingEvent"></EventsModal>
+  <EventsModal
+    v-if="isEditingEvent"
+    :type="'Edit'"
+    :event="editingEvent"
+    @close="closeModal"
+    v-model="isEditingEvent"
+  ></EventsModal>
```

Comment:
We can remove the closing tag and just close it off in the opening tag, like how you did for Event.